### PR TITLE
feat: added new youtube channel link(CodeNcode) in competitive programming tutorials

### LINKED ISSUE #1794: [Add] <Add new youtube tutorial link for competitive programming>
### Site URL

https://www.youtube.com/@codencode

### Description of the above provided site

This channel offers one of the best playlists for competitive programming, covering topics from beginner-level number theory to advanced topics like dynamic programming, graph theory, bit manipulations, and segment trees. The videos are explained in a very clear and easy-to-understand way.

### Category

Competitive Programming

### Sub Category

Tutorials

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by sravangorati2001**
Closes #1794 

## Changes proposed

Added new tutorial link in Category : Competitive Programming , Sub Category : Tutorials.

## Screenshots

Screenshot 2023-10-03 at 10 17 44 PM
Screenshot 2023-10-03 at 10 18 00 PM

## Note to reviewers
Hacktoberfest

### CODE REVIEW

File: database/competitive_programming/cp-tutorials.json [hunk 1]
```diff
@@ -30,5 +30,13 @@
     "category": "competitive-programming",
     "subcategory": "cp-tutorials",
     "language": "english"
+  },
+  {
+    "name": "CodeNCode",
+    "description": "This channel offers one of the best playlists for competitive programming, covering topics from beginner-level number theory to advanced topics like dynamic programming, graph theory, bit manipulations, and segment trees. The videos are explained in a very clear and easy-to-understand way.",
```
Comment (CBID2) [inline database/competitive_programming/cp-tutorials.json:36]: [proofreading] 
```suggestion
    "description": "This channel offers one of the best playlists for competitive programming, covering beginner topics like number theory and advanced topics such as dynamic programming, graph theory, bit manipulations, and segment trees. These concepts are explained in a very clear and easy-to-understand format.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @sravangorati2001! :) I pointed out some proofreading errors in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

**Review by CBID2 — APPROVED** [review]
Well done @sravangorati2001! @Anmol-Baranwal, it's your turn! 😄

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Pull request merged**